<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="widget">
   <widget class="QLabel" name="ghLogoLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="jstarLogoLabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>340</y>
      <width>220</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="terminalTabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>531</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTableView" name="qemuInstanceTable">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>581</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAFRL_RWWN_QEMU_Launcher">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_file_new_qemu_instance"/>
    <addaction name="separator"/>
    <addaction name="actionModify_Image_Contents"/>
    <addaction name="action_file_exit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_help_about"/>
   </widget>
   <addaction name="menuAFRL_RWWN_QEMU_Launcher"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_file_new_qemu_instance">
   <property name="text">
    <string>New QEMU Instance</string>
   </property>
  </action>
  <action name="action_help_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_file_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionModify_Image_Contents">
   <property name="text">
    <string>Modify Image Contents</string>
   </property>
   <property name="toolTip">
    <string>Opens GUI to copy items into guest image</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
